articles/quickstart/webapp/aspnet-owin/_includes/_login.md

```diff
@@ -1,14 +1,14 @@
 <!-- markdownlint-disable MD002 MD041 -->
 
-## Configure your application to use Auth0
+## Configure Your Application to Use Auth0
 
 [Universal Login](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security and the fullest array of features. This guide will use it to provide a way for your users to log in to your ASP.NET MVC 5 application.
 
 ::: note
 You can also create a custom login for prompting the user for their username and password. To learn how to do this in your application, follow the [Custom Login sample](https://github.com/auth0-samples/auth0-aspnet-owin-mvc-samples/tree/master/Samples/custom-login).
 :::
 
-### Install and configure the OpenID Connect middleware
+### Install and Configure the OpenID Connect Middleware
```

Comment:
Level 3 headings and below should be sentence case, so this one can be reverted.